backend/src/main/java/com/capstone/moneytree/service/impl/DefaultUserService.java

```diff
@@ -30,12 +34,17 @@
    private final ValidatorFactory validatorFactory;
    private final MoneyTreePasswordEncryption passwordEncryption;
    private static final Logger LOG = LoggerFactory.getLogger(DefaultUserService.class);
+   private final AmazonS3Service amazonS3Service;
+
+   @Value("${aws.profile.pictures.bucket}")
+   private String bucketName;
```

Comment:
I was never able to inject @ Values like that. You'll have to show it to me. It was always null for me. Nice!

Reply:
Maybe you were trying to inject inside one of the test, in which case null is normal. I'll explain: @value is a spring annotation and only works if the class is run with the spring runner class. In the test, it is currently the junit runner class that runs it and therefore it ignores (doesn't know) the spring annotation. FYI I changed it and will be merged when #165 is done.

Reply:
That might've been the case. Thanks!